aepp/config.py

```diff
@@ -6,10 +6,12 @@
     "tech_id": "",
```

Comment:
I see a file called configs.py and config.py, do we need both?

Reply:
Yes, `configs.py` provides a set of public functions intended to be used by the end-user like `aepp.configure` which you already know, and `config.py` is more for internal configuration objects that are not meant to be accessed directly.

Reply:
maybe we should rename these to make it more explicit

Reply:
They are not to be used by anyone, they are internal references.
I do not mind modifying it but they are used everywhere, which mean changing all references.